src/aideck/embeddings/ollama_provider.py

```
@@ -0,0 +1,573 @@
+            status_forcelist=[429, 500, 502, 503, 504],
+        )
+        
+        adapter = HTTPAdapter(max_retries=retry_strategy)
+        self.session.mount("http://", adapter)
+        self.session.mount("https://", adapter)
+    
+    def _initialize_sync(self):
+        """Initialize Ollama connection and model synchronously."""
+        try:
+            if self._check_ollama_health_sync():
+                print(f"Connected to Ollama server")
```

Ruff (0.13.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)